routes/routes.js

```diff
@@ -0,0 +1,10 @@
+const compassSchema = require('../models/compass');
+
+module.exports = {
```

Comment:
I think convention is to export an express Router and calling `app.use` on it. So that all the api mappings are in the routes file. See highlighted line here: https://github.com/phuston/audius78/blob/master/routes/workspace.js#L1

Reply:
Updated it to use express's router. For some reason I always preferred doing it the other way, although I guess using the router is more elegant/helps to separate code fully.